docs/source/ontology_intro.md

```diff
@@ -10,4 +10,4 @@ the representational primitives include information about their meaning
 and constraints on their logically consistent application. (Source:
 <http://tomgruber.org/writing/ontology-definition-2007.htm>)
 
-TODO extend
\ No newline at end of file
+[//]: # (TODO Extend)
```

Comment:
Should be extended at some point so I am keeping the comment but hiding it from the reader (it should not appear on the rendered docs).